src/js/big-play-button.js

```diff
@@ -44,9 +56,23 @@ class BigPlayButton extends Button {
       return;
     }
 
-    this.setTimeout(function() {
-      playToggle.focus();
-    }, 1);
+    if (playPromise) {
+      playPromise.then(() => playToggle.focus());
+    } else {
+      this.setTimeout(function() {
```

Comment:
maybe store this function since it is used above? or just use the shorthand again?